sentry-core/src/main/java/io/sentry/core/Hint.java

```diff
@@ -0,0 +1,4 @@
+package io.sentry.core;
+
+// Marker interface to provide extra information to callbacks
+public interface Hint {}
```

Comment:
@bruno-garcia what this interface should have? the methods from `Breadcrumb` and `Breadcrumb` should implement `Hint`?